optaplanner-examples/src/main/java/org/optaplanner/examples/common/persistence/StringDataGenerator.java

```diff
@@ -182,7 +182,7 @@ public StringDataGenerator addPart(String... partValues) {
         return this;
     }
 
-    /**
+    /*
```

Comment:
OK. It needs to look slightly different then - the warning was for the first line to end with a full stop. Will fix.